UefiPayloadPkg/UefiPayloadPkg.dsc

```diff
@@ -575,6 +575,7 @@
   MdeModulePkg/Application/UiApp/UiApp.inf {
     <LibraryClasses>
       NULL|MdeModulePkg/Library/DeviceManagerUiLib/DeviceManagerUiLib.inf
+      NULL|DasharoModulePkg/Library/DasharoSystemFeaturesUiLib/DasharoSystemFeaturesUiLib.inf

```

Comment:
Can this be done via DasharoModuleComponents.dsc.inc inclusion?

Reply:
I tried moving
```
  MdeModulePkg/Application/UiApp/UiApp.inf {
    <LibraryClasses>
      NULL|MdeModulePkg/Library/DeviceManagerUiLib/DeviceManagerUiLib.inf
      NULL|DasharoModulePkg/Library/DasharoSystemFeaturesUiLib/DasharoSystemFeaturesUiLib.inf
      NULL|MdeModulePkg/Library/BootManagerUiLib/BootManagerUiLib.inf
      NULL|MdeModulePkg/Library/BootMaintenanceManagerUiLib/BootMaintenanceManagerUiLib.inf
  }
```
to `.dsc.inc` and including it. Compiler doesn't complain about duplication. Specification says that sub-elements override previous definitions, so this might work, but needs to be tested.

Reply:
I thought rather about something like this:

```
  !include DasharoModulePkg/DasharoModuleComponents.dsc.inc
  MdeModulePkg/Application/UiApp/UiApp.inf {
    <LibraryClasses>
      NULL|MdeModulePkg/Library/DeviceManagerUiLib/DeviceManagerUiLib.inf
      NULL|MdeModulePkg/Library/BootManagerUiLib/BootManagerUiLib.inf
      NULL|MdeModulePkg/Library/BootMaintenanceManagerUiLib/BootMaintenanceManagerUiLib.inf
  }
```

Don;t know if it has a chance to work like that...

Reply:
We may leave it like that and change it later. Currently we do not have that much options.

Reply:
> Don't know if it has a chance to work like that...

I don't think it can, it probably needs to go into sub-elements to be picked up by the menu, which is why I just added `inf` file. Can figure this out later.